hydra-explorer/json-schemas/hydra-explorer-api.yaml

```diff
@@ -0,0 +1,195 @@
+      additionalProperties: false
+      required:
+        - scriptLanguage
+        - script
+      properties:
+        scriptLanguage:
+          type: string
+        script:
+          type: object
+          additionalProperties: false
+          required:
+            - cborHex
+            - description
+            - type
+          properties:
+            cborHex:
+              type: string
+            description:
+              type: string
+            type:
+              type: string
+              enum:
+                - SimpleScript
+                - PlutusScriptV1
+                - PlutusScriptV2
+      example:
+        {
+          "script": {
+              "cborHex": "8303018282051927168200581c0d94e174732ef9aae73f395ab44507bfa983d65023c11a951f0c32e4",
+              "description": "",
+              "type": "SimpleScript"
+          },
+          "scriptLanguage": "SimpleScriptLanguage"
+        }
+    TxOut:
+      type: object
+      description: |
+        A single transaction output
+      required:
+      - address
+      - value
+      additionalProperties: false
+      properties:
+        address:
+          $ref: "#/components/schemas/Address"
+        value:
+          $ref: "#/components/schemas/Value"
+        referenceScript:
+          $ref: "#/components/schemas/Script"
+        datumhash:
+          type: string
+        inlineDatum:
+          type: object
+          additionalProperties: true
+        inlineDatumhash:
+          type: string
+        datum:
+          type: string
+    PartyCommit:
+      type: object
+      properties:
+        txIn:
+          $ref: "#/components/schemas/TxIn"
+        txOut:
+          $ref: "#/components/schemas/TxOut"
+    Party:
+      type: object
+      description: |
+        The verification key for some Party in the Head protocol, uniquely identifying it.
+      additionalProperties: false
+      required:
+        - vkey
+      properties:
+        vkey:
+          type: string
+      example:
+        {
+            "vkey": "d0b8f28427aa7b640c636075905cbd6574a431aeaca5b3dbafd47cfe66c35043"
+        }
+    HeadMember:
+      type: object
+      properties:
+        party:
+          $ref: "#/components/schemas/Party"
+        onChainId:
+          $ref: '#/components/schemas/OnChainId'
+        commits:
+          type: array
+          items:
+            $ref: '#/components/schemas/PartyCommit'
+    HeadStatus:
+      type: string
+      enum:
+        - Initializing
+        - Aborted
+        - Open
+        - Closed
+        - Finalized
+    ContestationPeriod:
+      type: number
+      description: |
+        A contestation duration in seconds.
+      example: 60
+    HeadId:
+      type: string
+      description: |
+        A unique identifier for a Head, represented by a hex-encoded 16 bytes string.
+      example:
+        "820082582089ff4f3ff4a6052ec9d073"
+    HeadState:
+      type: object
+      properties:
+        headId:
```

Comment:
Should: `headId` should be `required` at least

Reply:
good catch!